gef-ui/server/controllers/mocks/index.js

```diff
@@ -277,6 +277,9 @@ const MockRequestUnissued = () => {
     user: {
       bank: { id: 'BANKID' },
       roles: ['MAKER'],
+      _id: 12345,
```

Comment:
v.minor but should be a string to be consistent

Reply:
done